flang/runtime/exceptions.cpp

```diff
@@ -81,6 +81,23 @@ uint32_t RTNAME(MapException)(uint32_t excepts) {
 // on some systems, e.g. Solaris, so omit object size comparison for now.
 // TODO: consider femode_t object size comparison once its more mature.
 
+// Check if the processor has the ability to control whether to halt or
+// continue execution when a given exception is raised.
+bool RTNAME(SupportHalting)(uint32_t except) {
+  except = RTNAME(MapException)(except);
+  int currentSet = fegetexcept(), flipSet, ok;
```

Comment:
Looks like you cannot use those on windows, the windows pre-merge buildbot is failing here:

```
C:\ws\src\flang\runtime\exceptions.cpp(88): error C3861: 'fegetexcept': identifier not found
C:\ws\src\flang\runtime\exceptions.cpp(90): error C3861: 'fedisableexcept': identifier not found
C:\ws\src\flang\runtime\exceptions.cpp(91): error C3861: 'fegetexcept': identifier not found
C:\ws\src\flang\runtime\exceptions.cpp(92): error C3861: 'feenableexcept': identifier not found
C:\ws\src\flang\runtime\exceptions.cpp(94): error C3861: 'feenableexcept': identifier not found
C:\ws\src\flang\runtime\exceptions.cpp(95): error C3861: 'fegetexcept': identifier not found
C:\ws\src\flang\runtime\exceptions.cpp(96): error C3861: 'fedisableexcept': identifier not found
```